yt_dlp/extractor/nzonscreen.py

```diff
@@ -0,0 +1,76 @@
+from .common import InfoExtractor
+from ..utils import (
+    float_or_none,
+    int_or_none,
+    qualities,
+    traverse_obj,
+    try_get,
+    url_or_none,
+)
+
+
+class NZOnScreenIE(InfoExtractor):
+    _VALID_URL = r'^https://www.nzonscreen.com/title/(?P<id>[^\?]+)'
+    _TESTS = [{
+        'url': 'https://www.nzonscreen.com/title/shoop-shoop-diddy-wop-cumma-cumma-wang-dang-1982',
+        'info_dict': {
+            'id': '726ed6585c6bfb30',
+            'display_id': 'shoop-shoop-diddy-wop-cumma-cumma-wang-dang-1982',
+            'ext': 'mp4',
+            'title': 'Monte Video - "Shoop Shoop, Diddy Wop"',
+            'playable_in_embed': False,
+            'thumbnail': r're:https://www\.nzonscreen\.com/content/images/.+\.jpg',
+            'duration': 158,
+        },
+        'params': {'skip_download': 'm3u8'},
+    }, {
+        'url': 'https://www.nzonscreen.com/title/shes-a-mod-1964?collection=best-of-the-60s',
+        'info_dict': {
+            'id': '3dbe709ff03c36f1',
+            'display_id': 'shes-a-mod-1964',
+            'ext': 'mp4',
+            'title': 'Ray Columbus - \'She\'s A Mod\'',
+            'playable_in_embed': False,
+            'thumbnail': r're:https://www\.nzonscreen\.com/content/images/.+\.jpg',
+            'duration': 130,
+        },
+        'params': {'skip_download': 'm3u8'},
+    }]
+
+    def _extract_formats(self, playlist):
+        quality = qualities(['lo_res', 'hd_res', 'hi_res'])
```

Comment:
Once hd_res is fixed, this should be adjusted according to which is better format